bundle/Resources/config/services.yml

```diff
@@ -18,3 +25,16 @@ services:
         parent: ezplatform.installer.db_based_installer
         tags:
             - {name: ezplatform.installer, type: netgen-media-clean}
+
+    netgen_site_installer.media_site_remote_clean:
+        class: Netgen\Bundle\SiteInstallerBundle\Installer\NetgenRemoteMediaSiteInstaller
+        public: false
+        calls:
+            - [ setInstallerDataPath, [ "%kernel.project_dir%/vendor/netgen/media-site-data/netgen-media-clean" ] ]
+            - [ setStoragePath, [ web/var/site/storage ] ]
+            - [ setRepository, [ "@ezpublish.api.repository" ] ]
+            - [ setFieldTypesToMigrate, [ "%netgen_site_installer.remote_media.field_types_to_migrate%" ] ]
+            - [ setExcludedFieldIdentifiers, [ "%netgen_site_installer.remote_media.excluded_field_identifiers%" ] ]
+        parent: ezplatform.installer.db_based_installer
+        tags:
+            - { name: ezplatform.installer, type: netgen-media-remote-clean }
```

Comment:
I presume demo variant is too complicated to migrate, so only `remote-clean` is available?

Reply:
Exactly, it might be implemented in the future, but for now this will be the starting point :)